### DISCUSSION TITLE
Can u guys add a blacklist channel command

### DISCUSSION THREAD

**Discussion by ghost** [SudhanPlayz/Discord-MusicBot#383] [Ideas]
blacklist a channel and on that channel ppl cant use play command and when u repeat the blacklist command on the same channel the channel will be back to normal and ppl can use play command there

**Comment by ghost** [comment]
good idea

**Comment by ItzSudhan** [comment]
Why does it really needed on music bot?

**Reply by ghost** [reply]
For just the person use the bot un one channel
